src/styles/components/_button.scss

```diff
@@ -83,7 +83,14 @@ button {
 	background-color: #ffe5e5;
 	&:active, &:focus {
 		background-color: #ffe5e5;
-	}	
+	}
+}
+
+.green {
+	background-color: #a7dc95;
```

Comment:
There are hard tab characters here

Looks like the whole file has them though, so nevermind